### PR TITLE
Add documentation for new-mail

### PR DESCRIPTION
**Pull request by SantiagoTorres**
This PR includes:

 - README.new-mail
 - muttrc.-new-mail
 - vimrc.new-mail

### CODE REVIEW

**Comment by flatcap** [comment]
Merged as 22ef61e on [feature/new-mail](https://github.com/neomutt/neomutt/tree/feature/new-mail)

File: README.new-mail [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+Introduction
+------------
+
+    This patch enables the new_mail_command setting, which can be used to execute a custom script (e.g., a notification handler) upon receiving a new mail. 
+
+    For example in Linux you can use (most distributions already provide notify-send):
+
+```
+    set new_command="notify-send --icon='/home/santiago/Pictures/mutt.png' 'New Email in %f' '%n new messages, %u unread.' &"
```
Comment (pierremoreau) [inline README.new-mail]: This should have been `set new_mail_command`, not `set new_command`

File: README.new-mail [hunk 2]
```diff
@@ -0,0 +1,48 @@
+new-mail-command Patch
+======================
+
+    Configure a custom command to be executed when a new email is received.
+
+Patch
+-----
+
+    To check if Mutt supports "new-mail-command", look for
+    "new-mail" in the mutt version.
```
Comment (pierremoreau) [inline README.new-mail]: Mutt seems to be supporting it, even if I do not get `new-mail` (or similar) in the output of `mutt -v`. Nor could I find any option to enable/disable it at compile time. Shouldn’t the only thing to look for be Mutt >= 1.6.2, as in the dependencies list?

Comment (flatcap) [inline README.new-mail]: @pierremoreau Thanks for your changes.

> Mutt seems to be supporting it, even if I do not get new-mail (or similar) in the output of mutt -v

The last release was a bit rushed and nothing quite agrees.
The next release (soon) will be a little more polished :-)

> Nor could I find any option to enable/disable it at compile time.

Generally, that's only been done for the larger, more invasive features.
new-mail is a small feature that's unlikely to cause problems.

**Comment by pierremoreau** [comment]
No problem! Waiting for the next release, and thank you for your work! :-)

**Pull request closed without merging**